next/components/atoms/Row/RowContent.tsx

```diff
@@ -7,11 +7,15 @@ type RowContentProps = {
   className?: string
 }
 
-const RowContent = ({ className, hover = true, children }: PropsWithChildren<RowContentProps>) => {
+const RowContent = ({
+  className = '',
+  hover = true,
+  children,
+}: PropsWithChildren<RowContentProps>) => {
   return (
     <div
       className={twMerge(
-        cx('relative flex grow items-center px-4 py-3 md:px-5 md:py-4', { group: hover }),
+        cx('relative flex grow items-center gap-x-4 px-4 py-3 md:px-5 md:py-4', { group: hover }),
```

Comment:
Why `gap-4` when in Figma it is 24px which is `gap-6`?
![image](https://github.com/user-attachments/assets/0cea1099-b423-439a-a80d-00b8243ff5cc)


Sidenote: Usually it's not needed to specify `gap-x-...` and it's cleaner to use `gap-...`.

Reply:
Also divider should have only 1px :)
![image](https://github.com/user-attachments/assets/c6c00c0e-6862-429d-aea3-575a5449edd9)